core/include/usModuleSettings.h

```diff
@@ -43,11 +45,15 @@ US_BEGIN_NAMESPACE
  * - \e US_AUTOLOAD_PATHS A ':' (Unix) or ';' (Windows) separated list of paths
  *   from which modules should be auto-loaded.
  *
+ * \deprecated Use FrameworkFactory::newFramework(std::map<std::string, std::string> configuration) to configure the framework.
```

Comment:
That sounds good. I'll create an issue to standardize the framework properties for theading, logging and storage along with an initial set of default framework property values. If you agree, I'll make this changes as a separate PR.

For storage configuration, can we not use the OSGi org.osgi.framework.storage property? This is a standard OSGi property used to define a Framework's persistent storage location. Its already used in this PR (usModuleContext.cpp - line 182). 

Could you elaborate on the trademark issues?